src/manifests_workflow/input_manifests.py

```diff
@@ -160,3 +171,28 @@ def add_to_cron(self, version: str) -> None:
             f.write(data)
 
         logging.info(f"Wrote {jenkinsfile}")
+
+    def add_to_versionincrement_workflow(self, version: str) -> None:
+        logging.info("Adding new version to the version increment workflow")
+        versionincrement_workflow_file = self.versionincrement_workflow()
+        yaml = ruamel.yaml.YAML()
+        yaml.explicit_start = True  # type: ignore
+        yaml.preserve_quotes = True  # type: ignore
+
+        with open(versionincrement_workflow_file) as f:
+            data = yaml.load(f)
+
+        version_entry = version.split(".")[0] + ".x"
+        for branch in data["jobs"]["plugin-version-increment-sync"]["strategy"]["matrix"]["branch"]:
+            if version_entry in branch:
```

Comment:
Why only check for `.x` branches?

Reply:
I dont think it is easy to check non-.x branch right?
We can definitely add a section to keep bumping tho.
But .x is the source of truth on the latest version.

Reply:
True, also we want to only version increment the `.x` branches, the plugin teams will anyhow use the `.x` as source of truth to cut the `MAJOR_VERSION.x` branch, by then its already incremented and we can run this workflow on crontab (or with some trigger), that keep checking the `.x` branch, compare with core and increment the version, this way the version is already incremented when the plugin team is ready to cut a release branch.
@gaiksaya

Reply:
I see. In case of patch releases however which track `major.minor` branch (not the `.x` branch) I believe we would need to add it manually then for now?

Reply:
Good point @gaiksaya let me handle the case for `major.minor` branch.

Reply:
Hey @gaiksaya @peterzhuamazon the change should now handle the `major.minor` branch, can you please review back again?